release-notes.html

```diff
@@ -212,9 +212,10 @@
         <li>Implemented the Security color for use within Leap.</li>
       </ul>
     </div>
-     <div class="col-container border-b-xs p-b-2-xs m-t-2-xs">
+  </div>
+  <div class="col-container border-b-xs p-b-2-xs m-t-2-xs">
     <div class="col col-10-md">
-      <p class="text-color-light">0.8.5</p>
+      <p class="text-color-light">0.8.6</p>
```

Comment:
Was this a fix of an existing error in the release notes?

Reply:
Yeeeep! It said 0.8.5 for that update as well as the previous, despite everywhere else in Leap reading 0.8.6. I just updated for clarity's sake (wanted to be clear I hadn't skipped a version # for this one, which is 0.8.7)

Reply:
Cool, everything otherwise looks good.